addons/api/tests/unit/serializers/worker-test.js

```diff
@@ -0,0 +1,98 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Serializer | worker', function (hooks) {
+  setupTest(hooks);
+  // TODO: turn on test when model PR has been merged - devon
+  test.skip('it serializes workers normally, without canonical tags type', function (assert) {
```

Comment:
How come the tests are being skipped?

Reply:
When i started this I didn't have the worker `models`, should have merged your PR into my branch or found a workflow where I had access.  I can get this done quickly.